frontend/src/helpers/api.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
+
+export default() => {
```

Comment:
This won't throw if the environment variable is not set, and will likely cause trouble if we want to deploy it behind a proxy later on. 

Also, axios will set the headers automatically.

```suggestion
const baseURL = (new URL('/api/v1', process.env.VUE_APP_BACKEND_URL || window.location.origin)).toString();

export default() => {
  return axios.create({
    baseURL,
  })
}
```